packages/office-ui-fabric-react/src/components/ComponentStatus/ComponentStatusPage.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
```

Comment:
Should this be under experiments/src/components so that it doesn't appear on the actual website?

Reply:
probably. Also, is it possible to have unknown as a status? instead of fail? not a big deal but might be a little softer.

Reply:
I suggested fail for the initial status to make it not be soft (e.g. so those badges are noticed). I'd prefer to have them invoke more of a reaction so that 1) it helps push the badges get initially calibrated and 2) to help component builders see and take into account the functionality the badges represent (instead of seeing a more soft "unknown" status which, I imagine, will give the builder the tendency to have an "oh 'unknown', that means I don't have to care right now" response)

Reply:
i think you're right @jspurlin .

Reply:
After talking with David, I added an unknown status to all components, so right now the default is unknown. This will help keep track which components have been audited and which ones are actually failing.